nodes/nomos-executor/executor/src/api/backend.rs

```diff
@@ -436,202 +437,229 @@ where
-            )
-            .route(
-                paths::MEMPOOL_ADD_TX,
-                routing::post(add_tx::<Tx, RuntimeServiceId>),
-            )
-            .route(
-                paths::MEMPOOL_ADD_BLOB_INFO,
-                routing::post(
-                    add_blob_info::<
-                        DaVerifiedBlobInfo,
-                        SamplingBackend,
-                        SamplingNetworkAdapter,
-                        SamplingRng,
-                        SamplingStorage,
-                        DaVerifierBackend,
-                        DaVerifierNetwork,
-                        DaVerifierStorage,
-                        ApiAdapter,
-                        RuntimeServiceId,
-                    >,
-                ),
-            )
-            .route(
-                paths::DISPERSE_DATA,
-                routing::post(
-                    disperse_data::<
-                        DispersalBackend,
-                        DispersalNetworkAdapter,
-                        DispersalMempoolAdapter,
-                        Membership,
-                        Metadata,
-                        RuntimeServiceId,
-                    >,
-                ),
-            )
-            .route(
-                paths::DA_GET_SHARES_COMMITMENTS,
-                routing::get(
-                    da_get_commitments::<
-                        DaStorageSerializer,
-                        DaStorageConverter,
-                        StorageAdapter,
-                        DaShare,
-                        RuntimeServiceId,
-                    >,
-                ),
-            )
-            .route(
-                paths::DA_GET_LIGHT_SHARE,
-                routing::get(
-                    da_get_light_share::<
-                        DaStorageSerializer,
-                        DaStorageConverter,
-                        StorageAdapter,
-                        DaShare,
-                        RuntimeServiceId,
-                    >,
-                ),
-            )
-            .route(
-                paths::DA_GET_SHARES,
-                routing::get(
-                    da_get_shares::<
-                        DaStorageSerializer,
-                        DaStorageConverter,
-                        StorageAdapter,
-                        DaShare,
-                        RuntimeServiceId,
-                    >,
-                ),
-            )
-            .route(
-                paths::DA_BALANCER_STATS,
-                routing::get(
-                    balancer_stats::<DaNetworkExecutorBackend<Membership>, RuntimeServiceId>,
-                ),
-            )
-            .route(
-                paths::DA_MONITOR_STATS,
-                routing::get(
-                    monitor_stats::<DaNetworkExecutorBackend<Membership>, RuntimeServiceId>,
-                ),
-            )
-            .with_state(handle);
+        let app =
```

Comment:
Why this change? is it because of formatting when introducing new generics?

Reply:
I believe it was the formatting. I think I did not introduce any change other than added generic params